src/graphics/circle.hpp

```diff
@@ -0,0 +1,64 @@
+#ifndef CIRCLE_HPP
+#define CIRCLE_HPP
+
+#include "axis.hpp"
+#include "drawable.hpp"
+#include "pad.hpp"
+
+#include <SFML/Graphics/CircleShape.hpp>
+#include <array>
+
+namespace graphic {
+
+class Circle : public Drawable {
```

Comment:
Why is there a separate class for the circle ?

Reply:
It's just a test class .. to be remove in time